packages/@cosmos-building-blocks/service/src/redis.ts

```diff
@@ -0,0 +1,109 @@
+import { Construct, IConstruct } from '@aws-cdk/core';
+import { CfnSubnetGroup, CfnReplicationGroup, CfnReplicationGroupProps } from '@aws-cdk/aws-elasticache';
+import { SecurityGroup, Port, IVpc, Connections, SubnetSelection, ISubnet } from '@aws-cdk/aws-ec2';
+
+export interface IRedis extends IConstruct {
+  /**
+   * Subnet Group created by construct using provided Subnets
+   */
+  readonly cacheSubnetGroup: CfnSubnetGroup;
+  /**
+   * Security Group assigned to Redis Cluster
+   */
+  readonly redisSecurityGroup: SecurityGroup;
+  /**
+   * Actual Redis Cluster
+   */
+  readonly redisReplicationGroup: CfnReplicationGroup;
+}
+
+/**
+ * Redis Cluster props. for the optional props not provided, following defaults will be used:
```

Comment:
minor thing, maybe a better description of props ?

Reply:
Added more details